src/lib/graphqlErrorHandler.js

```diff
@@ -2,9 +2,13 @@ import raven from "raven"
 import { assign } from "lodash"
 
 const blacklistHttpStatuses = [401, 403, 404]
-const shouldLogError = originalError => {
+
+export const shouldLogError = originalError => {
   if (originalError && originalError.statusCode) {
-    return !blacklistHttpStatuses.includes(originalError.statusCode)
+    return (
+      originalError.statusCode < 500 &&
```

Comment:
Makes sense, a 500 in an underlying service _should_ be reported in _that_ service's error reporting.

Reply:
Aye